[2023-11-07T02:52:05.855+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=check_S3_minio_bucket_v01/run_id=manual__2023-11-07T02:52:01.479538+00:00/task_id=1st_task permission to 509
[2023-11-07T02:52:05.966+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=check_S3_minio_bucket_v01/run_id=manual__2023-11-07T02:52:01.479538+00:00/task_id=1st_task permission to 509
[2023-11-07T02:52:06.112+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: check_S3_minio_bucket_v01.1st_task manual__2023-11-07T02:52:01.479538+00:00 [queued]>
[2023-11-07T02:52:06.172+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: check_S3_minio_bucket_v01.1st_task manual__2023-11-07T02:52:01.479538+00:00 [queued]>
[2023-11-07T02:52:06.175+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-11-07T02:52:06.248+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): 1st_task> on 2023-11-07 02:52:01.479538+00:00
[2023-11-07T02:52:06.275+0000] {standard_task_runner.py:57} INFO - Started process 23126 to run task
[2023-11-07T02:52:06.282+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'check_S3_minio_bucket_v01', '1st_task', 'manual__2023-11-07T02:52:01.479538+00:00', '--job-id', '316', '--raw', '--subdir', 'DAGS_FOLDER/check_S3_bucket.py', '--cfg-path', '/tmp/tmp69f8q0vm']
[2023-11-07T02:52:06.300+0000] {standard_task_runner.py:85} INFO - Job 316: Subtask 1st_task
[2023-11-07T02:52:06.454+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=check_S3_minio_bucket_v01/run_id=manual__2023-11-07T02:52:01.479538+00:00/task_id=1st_task permission to 509
[2023-11-07T02:52:06.464+0000] {task_command.py:416} INFO - Running <TaskInstance: check_S3_minio_bucket_v01.1st_task manual__2023-11-07T02:52:01.479538+00:00 [running]> on host c4630ce72677
[2023-11-07T02:52:06.728+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Toema' AIRFLOW_CTX_DAG_ID='check_S3_minio_bucket_v01' AIRFLOW_CTX_TASK_ID='1st_task' AIRFLOW_CTX_EXECUTION_DATE='2023-11-07T02:52:01.479538+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-07T02:52:01.479538+00:00'
[2023-11-07T02:52:06.754+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-07T02:52:06.763+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install minio urllib3']
[2023-11-07T02:52:06.795+0000] {subprocess.py:86} INFO - Output:
[2023-11-07T02:52:12.383+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-11-07T02:52:14.974+0000] {subprocess.py:93} INFO - Collecting minio
[2023-11-07T02:52:15.258+0000] {subprocess.py:93} INFO -   Downloading minio-7.1.17-py3-none-any.whl (78 kB)
[2023-11-07T02:52:15.488+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 78.3/78.3 kB 346.9 kB/s eta 0:00:00
[2023-11-07T02:52:15.506+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3 in /home/***/.local/lib/python3.8/site-packages (1.26.17)
[2023-11-07T02:52:15.517+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi in /home/***/.local/lib/python3.8/site-packages (from minio) (2023.7.22)
[2023-11-07T02:52:41.099+0000] {subprocess.py:93} INFO - Installing collected packages: minio
[2023-11-07T02:52:42.491+0000] {subprocess.py:93} INFO - Successfully installed minio-7.1.17
[2023-11-07T02:52:44.184+0000] {subprocess.py:93} INFO - 
[2023-11-07T02:52:44.190+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.3.1
[2023-11-07T02:52:44.192+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-11-07T02:52:45.532+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-11-07T02:52:45.829+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=check_S3_minio_bucket_v01, task_id=1st_task, execution_date=20231107T025201, start_date=20231107T025206, end_date=20231107T025245
[2023-11-07T02:52:46.096+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-07T02:52:46.226+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
